_sass/_myStyle.scss

```diff
@@ -49,17 +49,18 @@ body.frontpage {
     h3 {
         font-weight: bold;
     }
-    .guide-image {
-        float: right;
-        width: 200px;
-        height: 200px;
-        background: white no-repeat center center;
-        background-size: cover;
-        border: 10px solid white;
-    }
-    .guide {
-        margin: 40px 0;
-    }
+}
+
+.guide-image {
+    float: right;
+    width: 200px;
+    height: 200px;
+    background: white no-repeat center center;
+    background-size: cover;
+    border: 10px solid white;
+}
+.guide {
```

Comment:
this also affects `<body class=guide>`, pushing most of the pages on the site down a bit.